chia/plotting/manager.py

```diff
@@ -99,6 +147,7 @@ def needs_refresh(self) -> bool:
         return time.time() - self.last_refresh_time > float(self.refresh_parameter.interval_seconds)
 
     def start_refreshing(self):
+        self.load_cache()
```

Comment:
this sounds like something we would do periodically (`start_refreshing()`). If so, we would only need to load the cache the first time. Does this cause us to reload the cache every 2 minutes or so?

Reply:
This is only called once in `harvester_handshake` but i still moved the statements a bit so that its also safe if this gets called multiple times for some reason.